components/chainhook-sdk/src/indexer/bitcoin/tests.rs

```diff
@@ -1,5 +1,3 @@
-use bitcoincore_rpc::bitcoin::Script;
```

Comment:
The build warnings were bugging me, I hope this is okay

Reply:
Thanks!